include/bl_provision.rst

```diff
@@ -0,0 +1,22 @@
+.. _doc_bl_provision:
+
+Bootloader provision page
```

Comment:
Dubious about the verb "provision" vs "provisioned data" vs "provision page"

Reply:
We can discuss this in #1919 now.